iosApp/iosAppTests/iosAppTests.swift

```diff
@@ -596,12 +596,98 @@ class iosAppTests: XCTestCase {
         ApiHelper.shared.sendOutboundSmsMessage(conversationId: conversationInfo.conversationId, communicationId: conversationInfo.communicationId, message: receivedMessageText)
         testController.waitForTestExpectation()
         testController.verifyReceivedMessage(expectedMessage: receivedMessageText)
-
+        
         // Disconnect the conversation for the agent and disconnect the session.
         ApiHelper.shared.sendConnectOrDisconnect(conversationInfo: conversationInfo, connecting: false, wrapup: true)
         testController.disconnectMessenger()
     }
-
+    
+    func testStepUpAuthentication() {
```

Comment:
A possible issue with running this test with the full suite is that the suite runs tests in alphabetical order. This test would be ran very late and at that point, the auth token this test requires may expire before the test runs. Another test adds the prefix `AAA_` to the test name to mitigate this.

Reply:
Ok, I will make this change. Thank you.